config/unicorn.conf.rb

```diff
@@ -104,21 +98,48 @@
 
     if ENV["DISCOURSE_ENABLE_EMAIL_SYNC_DEMON"] == "true"
       server.logger.info "starting up EmailSync demon"
-      Demon::EmailSync.start
-      Signal.trap("SIGTSTP") do
-        STDERR.puts "#{Time.now}: Issuing stop to EmailSync"
-        Demon::EmailSync.stop
-      end
```

Comment:
This is removed for the same reasons as https://github.com/discourse/discourse/pull/27257/files#r1619723508